include/ui/input-text-model.hpp

```diff
@@ -25,7 +25,9 @@
 
 #include "core/memory-access.hpp"
 
-using length_t = unsigned ;
+namespace InputText{
+    using length_t = unsigned ;
+}
 
 class InputTextModel : QObject {
   Q_OBJECT
```

Comment:
If it can't be in the class for some Qt reason, I would have a `using length_t = InputText::length_t;` here so you don't have to type `InputText' everywhere